templates/resolv.conf.j2

```diff
@@ -1,19 +1,19 @@
-# {{ ansible_managed }}
+# {{ ansible_managed }}. role: resolv, template: resolv.conf.j2
```

Comment:
Useful to see which role and template this file comes from.
Cons : allready defined resolv.conf will have comment modifications / changed state